pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -853,3 +854,14 @@ func (r *ReconcileJenkinsBaseConfiguration) ensureBaseConfiguration(jenkinsClien
 
 	return reconcile.Result{Requeue: requeue}, err
 }
+
+func (r *ReconcileJenkinsBaseConfiguration) isOpenshiftEnabled() bool {
+	openshiftEnabled := false
+	if r.Configuration.Jenkins.Annotations[openshiftEnabledAnnotation] == "true" {
+		openshiftEnabled = true
+		r.logger.Info("-Openshift Mode-")
+	} else {
+		openshiftEnabled = false
+	}
+	return openshiftEnabled
```

Comment:
I think this function would be shortened to

```
func (r *ReconcileJenkinsBaseConfiguration) isOpenshiftEnabled() bool {
    return bool(r.Configuration.Jenkins.Annotations[openshiftEnabledAnnotation])
}
```

Or something like this

Reply:
let's not typecast, not a good practice.